langlib/lang.xml/src/main/ballerina/xml.bal

```diff
@@ -346,6 +489,15 @@ public isolated function 'map(xml<ItemType> x, @isolatedParam function(ItemType
 #
 # Each item is represented as a singleton value.
 #
+# ```ballerina
+# xml bookSet = xml `<book><title>Java</title></book><book><title>Ballerina</title></book>`;
+# xml bookTitles = xml ``;
+# 
+# bookSet.forEach(function (xml xmlItem) {
```

Comment:
```suggestion
# bookSet.forEach(isolated function (xml xmlItem) {
```

Reply:
Making this `isolated` would make it unable to access `bookTitles`.